tests/target/configs/enum_discrim_align_threshold/20.rs

```diff
@@ -0,0 +1,28 @@
+// rustfmt-enum_discrim_align_threshold: 20
+
+enum Standard {
+    A     = 1,
+    Bcdef = 2,
+}
+
+enum Mixed {
+    ThisIsAFairlyLongEnumVariantWithoutDiscrim,
+    A     = 1,
+    ThisIsAFairlyLongEnumVariantWithoutDiscrim2,
+    Bcdef = 2,
+}
+
+enum TooLong {
+    ThisOneHasDiscrimAaaaaaaaaaaaaaaaaaaaaaaaaaaa = 10,
+    A = 1,
+    Bcdef = 2,
+}

```

Comment:
An alternative would be to align all variants that are less than the threshold, rather than skip the whole enum:

```
enum TooLong {
    ThisOneHasDiscrimAaaaaaaaaaaaaaaaaaaaaaaaaaaa = 10,
    A     = 1,
    Bcdef = 2,
}
```

What do you think?